lib/Raven/Legacy.php

```diff
@@ -14,7 +16,7 @@ class Raven_Client extends \Raven\Client
 {
 }
 
-/**
+/** @noinspection PhpDeprecationInspection
```

Comment:
No. This tag means new class extends **deprecated** class. The new class (Raven_Compat) is still deprecated and will warn about it. 

I just like green mark.
![green](https://cloud.githubusercontent.com/assets/9131758/23095267/0d3e0716-f618-11e6-8ed5-c0fe631031a1.PNG)
![brown-gray](https://cloud.githubusercontent.com/assets/9131758/23095268/0d3f8b04-f618-11e6-8e59-8e0e17176054.PNG)

See #409